contrib/config/get_luet_root.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
+if [ $(id -u) -ne 0 ]
+  then echo "Please run the installer with sudo/as root"
+  exit
+fi
+
 set -ex
 export LUET_NOLOCK=true
 
-LUET_VERSION=0.9.24
+LUET_VERSION=$(curl -s https://api.github.com/repos/mudler/luet/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
```

Comment:
Can we keep a default version when the above command fails?

I'm thinking of it because the github API has a ratelimit, and if the user is above the threshold it would likely fail

Reply:
e.g.: